### ISSUE TITLE
support for scala

### ISSUE THREAD

**Issue by eranmizrahi** [phax/jcodemodel#44]
it will be nice if this library can be used to generate scala code...

**Comment by phax** [comment]
Good point :) But I really think this exceeds the scope for this library...
I'm not a Scala expert on the other hand, but I'm happy to receive some pull requests on this one :)

**Comment by phax** [comment]
Obviously interest is limited.
